DockerfileLeanFoundation

```diff
@@ -115,5 +121,13 @@ RUN git clone https://github.com/yhilpisch/dx.git && cd dx && \
 RUN git clone git://github.com/scikit-learn-contrib/py-earth.git && cd py-earth && \
     python setup.py install && cd .. && rm -irf py-earth
 
+# Install fastText
+RUN git clone https://github.com/facebookresearch/fastText.git && cd fastText && \
```

Comment:
Let's host a fixed copy of this on the CDN so it's deterministic. We can name the copy: `fastText-masterHash.zip`.
https://cdn.quantconnect.com/fastText/fastText-master-9479e4e.zip